src/components/ec-inline-input-field/ec-inline-input-field.spec.js

```diff
@@ -139,5 +142,29 @@ describe('EcInlineInputField', () => {
         expect(wrapper.findByDataTest('ec-inline-input-field-loading').exists()).toBeTruthy();
       });
     });
+
+    describe('when component is copiable', () => {
+      it('should render as expected', async () => {
+        const wrapper = mountInlineInputField({ isEditable: false, isCopiable: true });
+        await wrapper.vm.$nextTick();
+
+        expect(wrapper.element).toMatchSnapshot();
+        expect(wrapper.findByDataTest('ec-inline-input-field-copy').exists()).toBeTruthy();
+      });
+
+      it('should trigger copy method after clicking on the copy button', async () => {
+        const wrapper = mountInlineInputField(
```

Comment:
It is a little strange that for the previous test we represent the object in a single line, and in this one the object is represented in several lines.

Reply:
I had a similar thought